### PR TITLE
flip module.exports and window check

### PR DESCRIPTION
**Pull request by hunterjm**
per Issue #13, this will fix this lib on electron and nw.js clients.

### CODE REVIEW

File: src/s2geometry.js [hunk 1]
```diff
@@ -484,7 +484,7 @@ S2.S2Cell.nextKey = S2.nextKey = function (key) {
   return S2.stepKey(key, 1);
 };
 
-})('undefined' !== typeof window ? window : module.exports);
+})('undefined' !== typeof module.exports ? module.exports : window);
```
Comment (davidmurdoch) [inline src/s2geometry.js]: Won't this break in browsers since `module` is undefined?

**Comment by coolaj86** [comment]
v1.2.8 is yours for the testing (works in node, please try browser and node-webkit as applicable on your end)

File: src/s2geometry.js (hunk 1 above)
Comment (coolaj86) [inline src/s2geometry.js]: Yep, but in the version I published it only checks module

**Comment by hunterjm** [comment]
Verified working in electron on 1.2.8.

**Pull request merged**